extension/controller/backgroundScript.js

```diff
@@ -9,13 +10,47 @@
  * extension's background process.
  */
 
+const scrubSentryEvent = ev => {
+
+  /*
+   * scrub extension installation id
+   * urls are also stripped on backend based on global PII rule for Mozilla org
+   */
+  const removeMozId = s => s.replace(/moz-extension:\/\/[^/?#]*(.*)/gm, "$1");
```

Comment:
added http/https, but I do not observe any actual URLs there except moz-extension:// ones

Reply:
also added deleting `event.request.url` completely, it wasn't very useful anyway

Reply:
Nit: Good practice would be to have an early

```js
if (!ev.request.url.startsWith("moz-extension://") {
// Either don't send this sentry report at all since the error is not part of the extension
// or at least do
ev.request.url = null;
}
```